lasagne/random.py

```diff
@@ -0,0 +1,36 @@
+"""
+A module with a package-wide random number generator,
+used for weight initialization and seeding noise layers.
+This can be replaced by a `numpy.random.RandomState` instance with a
+particular seed to facilitate reproducibility.
+"""
+
+import numpy as np
+
+
+_rng = np.random
+
+
+def get_rng():
+    """Get the package-level random number generator.
+
+    Returns
+    -------
+    :class:`numpy.random.RandomState` instance
+        The :class:`numpy.random.RandomState` instance passed to the most
+        recent call of :func:`set_rng`, or `numpy.random` if :func:`set_rng` has
+        never been called.
```

Comment:
`has` needs to be moved into the next line, otherwise it's too long (that's what Travis is complaining above).

While we're at it, `numpy.random` needs double apostrophes as in [my suggestion](https://github.com/Lasagne/Lasagne/pull/343#discussion_r35886511). Same for the module docstring above. The others are correct! (It takes a while to get used to Sphinx' ReST markup.)